common/reactive/src/main/java/io/helidon/common/reactive/BaseProcessor.java

```diff
@@ -76,6 +76,8 @@ public final void onNext(T item) {
             } catch (Throwable ex) {
                 onError(ex);
             }
+        } else {
```

Comment:
I would consider changing the if statement to remove the negation, e.g. (pseudo):
```
if (sub.closed()) {
  throw ISE;
} else {
  hookOnNext
}
```

As analyzing if statements with negation and else branch is a brain twister

Reply:
Will change, don't have a preference either way.